tests/unit/backend/test_interface.py

```diff
@@ -101,3 +101,17 @@ def test_exchange_async(self):
             pass
         self.assertTrue(self.backend.mock.exchange_async_raw.called)
         self.assertEqual(self.backend.mock.exchange_async_raw.call_args, ((expected, ), ))
+
+
+class TestBackendInterfaceLogging(TestCase):
+
+    def test_log_apdu(self):
+        self.firmware = _Firmware("nanos", "2.0.1", "other")
+        test_file = Path("/tmp/test_log_file.log").resolve()
```

Comment:
If you want to clean up all created file / directory after you test is done, you can use the `tempfile` buildin library:

```python
with tempfile.TemporaryDirectory() as td:
    test_file = Path(td.name) / "test_log_file.log"
   [...]
```

The context manager will automatically remove the directory after the tests.